plugins/animation_to_java.js

```diff
@@ -0,0 +1,84 @@
+(function () {
+  var menuButton;
+
+  Plugin.register("animation_to_java", {
+    title: "Animation to Java Converter",
+    author: "MG",
+    description:
+      "Converts Blockbench animations to Java code for the new 1.19 keyframe system",
+    icon: "fa-cube",
+    version: "1.0.0",
+    variant: "both",
+    tags: ["Minecraft: Java Edition"],
+    onload() {
+      menuButton = new Action("export_to_java", {
```

Comment:
The ID should include animation. IDs are global so they should be specific to reduce the risk of conflict.